web/docker/README.md

```diff
@@ -0,0 +1,99 @@
+# Docker
+Docker provides a way to run applications securely isolated in a container,
+packaged with all its dependencies and libraries.
+
+To see how you can run CodeChecker server in docker read the following
+sections.
+
+## Build docker image
+You can create a docker image by using the following command:
+```bash
+docker build -t codechecker-server:latest .
```

Comment:
I'm thinking to configure all the commands so they can be run from the project root.